Plugins/SequencePlugin/Source/SequencePlugin/Private/ABI/ABI.cpp

```diff
@@ -51,19 +48,25 @@ TDynamicABIData ABI::String(FString Input)
 	return TDynamicABIData(Arr);
 }
 
-FUnsizedData ABI::Encode(FString Signature, TArray<ABIEncodeable*> Arr)
+FUnsizedData ABI::Encode(FString Signature, TArray<TSharedPtr<ABIElement>> Args)
```

Comment:
I noticed in one of the test files that this broke the integration code. Is there a way we can mark the old version deprecated and nudge integrators towards the new version so that we don't break existing integrations?

I know you had mentioned a risk of memory leaks... Is this relevant here?

Reply:
I was committing a change that adds an override function for ABI::Encode for the older version. Let me know what you think. It works on my end, although not the most memory-efficient thing, but that shouldn't be a problem for that specific call in project integrations. Then we can change the Docs to use the new function and put a note in the changelogs to switch to the new one.

Reply:
I think keeping the Encoding backwards compatible should be possible 👍 I'll take a look at that coming up